examples/operations/public/js/user-accounts.js

```diff
@@ -1,171 +1,172 @@
     var data = {
       orig_nick: orig_nick,
-      email: $( '#user_email' ).val(),
-      name: $( '#user_name' ).val()
+      email: $('#user_email').val(),
+      name: $('#user_name').val()
     }
-    $.post( '/auth/update_user', data, function ( instance ) {
+    $.post('/auth/update_user', data, function (instance) {
       cleanMessages()
-      if ( !instance.ok ) {
-        $( '#errorMsg' ).text( 'Error: ' + instance.why )
+      if (!instance.ok) {
+        $('#errorMsg').text('Error: ' + instance.why)
         return
       }
       else {
-        $( '#msg' ).text( 'User updated successfully.' )
+        $('#msg').text('User updated successfully.')
       }
-      showAccount( instance )
-    } )
+      showAccount(instance)
+    })
     return false
-  } )
+  })
 
-  $( '#execute_reset' ).click( function () {
+  $('#execute_reset').click(function () {
     var data = {
-      token: $( '#reset_token' ).val(),
-      password: $( '#new_password' ).val()
+      token: $('#reset_token').val(),
+      password: $('#new_password').val()
     }
-    $.post( '/auth/execute_reset', data, function ( instance ) {
+    $.post('/auth/execute_reset', data, function (instance) {
       cleanMessages()
-      if ( !instance.ok ) {
-        $( '#errorMsg' ).text( 'Error: ' + instance.why )
+      if (!instance.ok) {
+        $('#errorMsg').text('Error: ' + instance.why)
         return
       }
       else {
-        $( '#msg' ).text( 'Password updated successfully.' )
+        $('#msg').text('Password updated successfully.')
       }
       showLogin()
-    } )
+    })
     return false
-  } )
+  })
 
-  $( '#load_reset' ).click( function () {
+  $('#load_reset').click(function () {
     var data = {
-      token: $( '#reset_token' ).val()
+      token: $('#reset_token').val()
     }
-    $.post( '/auth/load_reset', data, function ( instance ) {
+    $.post('/auth/load_reset', data, function (instance) {
       cleanMessages()
-      if ( !instance.ok ) {
-        $( '#errorMsg' ).text( 'Error: ' + instance.why )
+      if (!instance.ok) {
+        $('#errorMsg').text('Error: ' + instance.why)
         return
       }
       else {
-        $( '#msg' ).text( 'Reset token ' + ( instance.active ? 'active' : 'not active' ) + '. User: ' + instance.nick )
+        $('#msg').text('Reset token ' + (instance.active ? 'active' : 'not active') + '. User: ' + instance.nick)
       }
-    } )
+    })
     return false
-  } )
+  })
 
-  $( '#logout' ).click( function () {
+  $('#logout').click(function () {
     cleanMessages()
-    $.post( '/auth/logout', {}, function ( instance ) {
+    $.post('/auth/logout', {}, function (instance) {
       cleanMessages()
-      $( '#msg' ).text( 'Logout successfully.' )
-      showLogin( instance )
-    } )
-  } )
+      $('#msg').text('Logout successfully.')
+      showLogin(instance)
+    })
+  })
 
-  $( '#create_reset' ).click( function () {
+  $('#create_reset').click(function () {
```

Comment:
@mihaidma 

This will not work if we are not sending the token anymore, we need to fix it somehow.